code/client/src/pages/Show.jsx

```diff
@@ -571,9 +570,8 @@ const Show = () => {
             <Label><Hint>To</Hint></Label>
             <AddressInput
               addressValue={transferTo}
-              setAddressCallback={(value) => setTransferTo(value)}
+              setAddressCallback={setTransferTo}
```

Comment:
Would that cause a warning "modifying another component's state inside a component"?

Reply:
I haven't seen this warning. But it is a call back to set the value in this component, and this component's value is being supplied as prop to the `AddressInput` component.